### PR TITLE
upgrade to kafka 2X

### PR DESCRIPTION
**Pull request by marky-mark**
* kafka 2X
* some plugins

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ We created Remora after spending some time using Linkedin's [burrow](https://git
 
 ### Dependencies
 
-The latest release of [Remora](https://github.com/zalando-incubator/remora) supports [Apache Kafka](http://kafka.apache.org/) 0.10.2.1 and 1.0.0.
+The latest release of [Remora](https://github.com/zalando-incubator/remora) supports [Apache Kafka](http://kafka.apache.org/) 0.10.2.1, 1.0.0 and 1.1.1.
```
Comment (imduffy15) [inline README.md]: and 2.0.0?

File: basic-example/docker-compose.yml [hunk 2]
```diff
@@ -1,19 +1,23 @@
...
-    hostname: kafka
+    image: wurstmeister/kafka:2.11-2.0.0
+    ports:
+      - target: 9092
+        published: 9092
+        protocol: tcp
+        mode: host
     environment:
-      KAFKA_ADVERTISED_HOST_NAME: kafka
+      KAFKA_ADVERTISED_HOST_NAME: $LOCAL_IP
```
Comment (imduffy15) [inline basic-example/docker-compose.yml]: This didn't work for me, I got errors about the `KAFKA_ADVERTISED_HOST_NAME` and `KAFKA_ADVERTISED_PORT` being deprecated as documented over here https://github.com/wurstmeister/kafka-docker#listener-configuration

`KAFKA_LISTENERS: PLAINTEXT://kafka:9092` seemed to work just fine.

**Comment by imduffy15** [comment]
👍

File: basic-example/docker-compose.yml (hunk 2 above)
Comment (marky-mark) [reply basic-example/docker-compose.yml]: If i exchanged LOCAL_IP with the local machines ip it worked. i'll give your suggestion a shot, couldn't get it working yesterday.

Thanks

**Comment by marky-mark** [comment]
Few little hacks @imduffy15 but got it working.

File: basic-example/docker-compose.yml (hunk 2 above)
Comment (marky-mark) [reply basic-example/docker-compose.yml]: Yea the issue was when contacting kafka from the outside 

`./kafka-console-consumer.sh --bootstrap-server localhost:9092 --topic test-0
[2018-10-09 09:47:17,810] WARN [Consumer clientId=consumer-1, groupId=console-consumer-17355] Error connecting to node kafka:9092 (id: 1001 rack: null) (org.apache.kafka.clients.NetworkClient)
java.io.IOException: Can't resolve address: kafka:9092
	at org.apache.kafka.common.network.Selector.doConnect(Selector.java:235)`

**Comment by marky-mark** [comment]
👍

**Comment by marky-mark** [comment]
👍

**Comment by marky-mark** [comment]
@imduffy15 need another thumbs up

File: basic-example/docker-compose.yml (hunk 2 above)
Comment (imduffy15) [reply basic-example/docker-compose.yml]: Hacky fix - add kafka to your `/etc/hosts` for 127.0.0.1 and everything resolves.... we can leave it aside for this PR. Running kafka in docker always has some caveats.

**Comment by imduffy15** [comment]
👍

**Pull request merged**